Droid/Playing/PollingService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Xamarin.Forms;
+
+namespace MeNext.Droid
+{
+	/// <summary>
+	/// Class which handles background polling for Android
+	/// </summary>
+	[Service]
+	public class PollingService : Service
```

Comment:
It seems like we can do something like forms tasks in the shared code to start background jobs there...